subsys/net/lib/nrf_cloud/coap/src/dtls.c

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/net/coap.h>
+#include <zephyr/net/socket.h>
+#include <modem/lte_lc.h>
+#include <modem/modem_key_mgmt.h>
+#if defined(CONFIG_MODEM_INFO)
+#include <modem/modem_info.h>
+#endif
+#include <nrf_socket.h>
+#include <nrf_modem_at.h>
+
+#include "dtls.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(dtls, CONFIG_NRF_CLOUD_COAP_LOG_LEVEL);
+
+static int sectag = CONFIG_NRF_CLOUD_COAP_SEC_TAG;
+static bool dtls_cid_active;
+static bool mfw_ge_version_2;
```

Comment:
what does the "ge" stand for?

Reply:
Greater than or equal, which is of course from everyone's everyday language, FORTRAN.

Reply:
Removed this flag.